tests/app/views/test_brief_response.py

```diff
@@ -324,6 +325,110 @@ def test_day_rate_should_be_less_than_service_max_price(self, live_dos_framework
+
+    def setup_existing_brief_response(self, brief_response_data):
+        res = self.create_brief_response(dict(brief_response_data, **{
+            'briefId': self.brief_id,
+            'supplierId': 0,
+        }))
+        assert res.status_code == 201
+
+        self.brief_response_id = json.loads(res.get_data(as_text=True))['briefResponses']['id']
+
+    def submit_brief_response(self, brief_response_id):
+        return self.client.post(
+            '/brief-responses/{}/submit'.format(brief_response_id),
+            data=json.dumps({
+                'updated_by': 'test@example.com',
+            }),
+            content_type='application/json'
+        )
+
+    @given(example_listings.brief_response_data())
+    def test_valid_draft_brief_response_can_be_submitted(self, live_dos_framework, brief_response_data):
+        self.setup_existing_brief_response(brief_response_data)
+
+        with freeze_time('2016-9-28'):
+            res = self.submit_brief_response(self.brief_response_id)
+        assert res.status_code == 200
+
+        brief_response = json.loads(res.get_data(as_text=True))['briefResponses']
+
+        assert brief_response['status'] == 'submitted'
+        assert brief_response['submittedAt'] == '2016-09-28T00:00:00.000000Z'
+
+    @given(example_listings.brief_response_data())
+    def test_submit_brief_response_creates_an_audit_event(self, live_dos_framework, brief_response_data):
+        self.setup_existing_brief_response(brief_response_data)
+
+        res = self.submit_brief_response(self.brief_response_id)
+
+        with self.app.app_context():
+            audit_events = AuditEvent.query.filter(
+                AuditEvent.type == AuditTypes.submit_brief_response.value
+            ).all()
+
+        assert len(audit_events) == 1
+        assert audit_events[0].data == {
+            'briefResponseId': self.brief_response_id
+        }
+
+    def test_submit_brief_response_that_doesnt_exist_will_404(self):
+        res = self.submit_brief_response(100)
+        assert res.status_code == 404
+
+    @given(example_listings.brief_response_data())
+    def test_can_not_submit_a_brief_response_that_already_been_submitted(self, live_dos_framework, brief_response_data):
+        self.setup_existing_brief_response(brief_response_data)
+
+        res = self.submit_brief_response(self.brief_response_id)
+        assert res.status_code == 200
+
+        repeat_res = self.submit_brief_response(self.brief_response_id)
+        assert repeat_res.status_code == 400
+
+        data = json.loads(repeat_res.get_data(as_text=True))
+        assert data == {'error': 'Brief response must be a draft'}
+
+    @given(example_listings.brief_response_data())
+    def test_can_not_submit_a_brief_response_for_a_framework_that_is_not_live(
+        self, live_dos_framework, brief_response_data
+    ):
+        self.setup_existing_brief_response(brief_response_data)
+
+        with self.app.app_context():
```

Comment:
It might be neater here to have an `expired_dos_framework` fixture to pass in, rather than passing in the live one and then updating it?